### DISCUSSION TITLE
Questions about wiring the adapter v5 to the ebus

### DISCUSSION THREAD

**Discussion by blastrock** [john30/ebusd#1079] [Q&A] [answered]
Hi there,

I am planning on buying an adapter v5 but I am unsure about whether it will work.

First of all, I don't know if this is common, but my thermostat is connected to my heater with only 2 wires, which carry both power and ebus. So there is 24V power on these wires. Will the adapter v5 work with that, or will it fry?

And second, I plan to use the adapter in Wi-Fi. The page says that it needs to be powered externally, with USB or with the pins. Can't it take power from the ebus wires directly?

Thanks for your input,
Philippe

**Comment by Koky05** [comment, marked as answer]
It could be connected to same wire as youre termostat parallel to it, but you need to connect your eBus externaly. It could not.take power from that wires.

**Reply by blastrock** [reply]
Ok thanks for the answer! I'll find a way to power it externally then.
